components/datetime/src/skeleton/helpers.rs

```diff
@@ -564,26 +565,38 @@ impl DateTimeFormatterOptions {
                     &components_bag,
                     false,
                 ) {
-                    BestSkeleton::AllFieldsMatch(p) => Some(p),
-                    _ => None,
+                    BestSkeleton::AllFieldsMatch(p) => p,
+                    _ => {
+                        // Build a last-resort pattern that contains all of the requested fields.
+                        // This is NOT in the CLDR standard! Should be using Append Items.
```

Comment:
question: should we file and link to an issue for this?

Reply:
Added reference to https://github.com/unicode-org/icu4x/issues/586